src/lib.ml

```diff
@@ -1179,6 +1179,13 @@ let report_lib_error ppf (e : Error.t) =
       Lib_name.pp_quoted not_vlib.name
       Lib_name.pp_quoted impl.name
 
+let () =
```

Comment:
Why do we need to report the error here? Don't we already do it in report_errors?